Configuration/TCA/tx_igldapssoauth_config.php

```diff
@@ -1,4 +1,6 @@
 <?php
+$sitesField = version_compare(TYPO3_version, '9.0', '>=') ? 'sites,' : '';
```

Comment:
my code uses the comma afterwards...

Reply:
Oh boy. I have cleaned up the commits into just two and fixed this error.